db/submission_test.go

```diff
@@ -59,3 +59,54 @@ func (this *DBTests) DBTestRemoveSubmission(test *testing.T) {
         }
     }
 }
+
+func (this *DBTests) DBTestFetchAttempts(test *testing.T) {
+    ResetForTesting();
+    defer ResetForTesting();
+
+    assignment := MustGetTestAssignment()
+
+    // User has submissions
```

Comment:
Try to make comments either a complete sentence or some Title Cased Short Comment.